modules/@apostrophecms/asset/lib/webpack/apos/webpack.config.js

```diff
@@ -28,7 +28,7 @@ module.exports = ({
     optimization: {
       minimize: process.env.NODE_ENV === 'production'
     },
-    devtool: 'eval-source-map',
+    devtool: 'source-map',
```

Comment:
Without this change Vue generates eval() calls. I did a git log search and was unable to find any evidence that we ever tried `source-map` before trying `eval-source-map`, which I believe is for really jurassic bc when debugging.